Packages/com.unity.inputsystem/InputSystem/Plugins/XR/XRSupport.cs

```diff
@@ -316,108 +317,8 @@ public static void Initialize()
             InputSystem.RegisterLayout<XRController>();
 
             InputSystem.onFindLayoutForDevice += XRLayoutBuilder.OnFindLayoutForDevice;
-
```

Comment:
Discussed this with the team. In the end, agreed we should leave the registrations in as long as we have the device classes still around in the package. Both should be nuked together.

ATM, if you happen to have a project that does indeed reference these classes, then with the change here, you wouldn't get a compile error but the project likely wouldn't work either.

Think the chances of this making a difference are slim. In practice, we likely wouldn't break anyone by just nuking the whole thing now. But for the sake of limiting breakage, let's just keep it and then remove it en bloc when we break the API.

Reply:
I have a tested change that does remove the define. however that define is a manual "I absolutely need everything off" and isnt intended to be used outside a major problem, in which case the define doesnt quite do what it's supposed to do.